ktor-server/jvm/src/io/ktor/server/application/ApplicationCall.kt

```diff
@@ -19,22 +20,35 @@ public interface ApplicationCall {
     public val application: Application
 
     /**
-     * Client request
+     * Client request.
      */
     public val request: ApplicationRequest
 
     /**
-     * Server response
+     * Server response.
      */
     public val response: ApplicationResponse
 
     /**
-     * Attributes attached to this instance
+     * Attributes attached to this instance.
      */
     public val attributes: Attributes
 
     /**
-     * Parameters associated with this call
+     * Parameters associated with this call.
      */
     public val parameters: Parameters
+
+    /**
+     * Callback that will be executed after the call finishes its execution.
+     *
+     * @param exception optional exception that happened before the call has finished
```

Comment:
exception that can take place before the call has finished. 

(Also this is not entirely clear...how can an exception be optional?)